### PR TITLE
uuid, uuidd, uuidgen: add Polish translation

### PR DESCRIPTION
**Pull request by chanibal**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pl/linux/uuid.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# uuid
+
+> Twórz i dekoduj uniwersalne identyfikatory (UUID).
+> Zobacz też `uuidgen`.
+> Więcej informacji: <https://manned.org/uuid>.
+
+- Stwórz UUIDv1 (oparte o zegar systemowy i - jeśli dostępne - adres sprzętowy):
```
Comment (EmilySeville7cfg) [inline pages.pl/linux/uuid.md:7]: The following fix can be applied after the [original](https://github.com/tldr-pages/tldr/blob/main/pages/linux/uuid.md) page is updated:

```suggestion
- Stwórz UUIDv1 oparte o zegar systemowy i - jeśli dostępne - adres sprzętowy:
```

Braces are redundant.

File: pages.pl/linux/uuid.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+- Stwórz UUIDv4 i zapisz do pliku:
+
+`uuid -v {{4}} -o {{ścieżka/do/pliku}}`
+
+- Stwórz UUIDv5 (oparty o podaną nazwę obiektu) w przestrzeni nazw:
+
+`uuid -v {{5}} ns:{{DNS|URL|OID|X500}} {{nazwa_obiektu}}`
+
+- Dekoduj podany UUID:
```
Comment (EmilySeville7cfg) [inline pages.pl/linux/uuid.md:31]: The following fix can be applied after the [original](https://github.com/tldr-pages/tldr/blob/main/pages/linux/uuid.md) page is updated:

```suggestion
- [d]ekoduj podany UUID:
```

Code descriptions can be changed to provide mnemonics for short options.

File: pages.pl/linux/uuid.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+> Twórz i dekoduj uniwersalne identyfikatory (UUID).
+> Zobacz też `uuidgen`.
+> Więcej informacji: <https://manned.org/uuid>.
+
+- Stwórz UUIDv1 (oparte o zegar systemowy i - jeśli dostępne - adres sprzętowy):
+
+`uuid`
+
+- Stwórz UUIDv4 (losowy):
```
Comment (EmilySeville7cfg) [inline pages.pl/linux/uuid.md:11]: The following fix can be applied after the [original](https://github.com/tldr-pages/tldr/blob/main/pages/linux/uuid.md) page is updated:

```suggestion
- Stwórz UUIDv4 losowy:
```

Braces are redundant.

File: pages.pl/linux/uuid.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+`uuid -v {{4}}`
+
+- Stwórz wiele UUIDv4 na raz:
+
+`uuid -v {{4}} -n {{ilość_uuid}}`
+
+- Stwórz UUIDv4 w konkretnym formacie:
```
Comment (EmilySeville7cfg) [inline pages.pl/linux/uuid.md:19]: The following fix can be applied after the [original](https://github.com/tldr-pages/tldr/blob/main/pages/linux/uuid.md) page is updated:

```suggestion
- Stwórz UUIDv4 w konkretnym [F]ormacie:
```

Code descriptions can be changed to provide mnemonics for short options.

File: pages.pl/linux/uuid.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+`uuid -v {{4}} -F {{BIN|STR|SIV}}`
+
+- Stwórz UUIDv4 i zapisz do pliku:
+
+`uuid -v {{4}} -o {{ścieżka/do/pliku}}`
+
+- Stwórz UUIDv5 (oparty o podaną nazwę obiektu) w przestrzeni nazw:
```
Comment (EmilySeville7cfg) [inline pages.pl/linux/uuid.md:27]: The following fix can be applied after the [original](https://github.com/tldr-pages/tldr/blob/main/pages/linux/uuid.md) page is updated:

```suggestion
- Stwórz UUIDv5 oparty o podaną nazwę obiektu w przestrzeni nazw:
```

Braces are redundant.

File: pages.pl/linux/uuidgen.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+`uuidgen --random`
+
+- Stwórz UUIDv1 oparty o aktualny czas:
+
+`uuidgen --time`
+
+- Stwórz UUIDv5 z nazwy i prefiksu przestrzeni nazw:
+
+`uuidgen --sha1 --namespace {{@dns|@url|@oid|@x500}} --name {{nazwa_obiektu}}`
```
Comment (EmilySeville7cfg) [inline pages.pl/linux/uuidgen.md:17]: The following fix can be applied after the [original](https://github.com/tldr-pages/tldr/blob/main/pages/linux/uuidgen.md) page is updated:

```suggestion
`uuidgen --sha1 --namespace @{{dns|url|oid|x500}} --name {{nazwa_obiektu}}`
```

Common placeholder parts can be moved out of them.

**Review by EmilySeville7cfg — APPROVED** [review]
No changes are requested because all pages are synced with the original ones even the last ones requires modifications.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**